drop.js

```diff
@@ -0,0 +1,5 @@
+export function drop(num,arr){
+	return arr.filter(function(item,index){
```

Comment:
* przeformatować kod
* nawias przy ifach

Generalnie to zadziala ale jest to bardzo nieoptymalny sposób.
może pomyśl jak wyciąć tablicę z tablicy ???

Tutaj możesz znaleźć listę funkcji na pewno znajdziesz coś pomocnego:
[Array](https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Array)